packages/embed/src/utils/load-options-from-attributes.spec.ts

```diff
@@ -87,6 +87,10 @@ describe('load-options-from-attributes', () => {
       expect(transformAttributeValue('a, b', 'array')).toEqual(['a', 'b'])
     })
 
+    it('should transform string with escaped comas as array', () => {
+      expect(transformAttributeValue('a, b, c\\,d', 'array')).toEqual(['a', 'b', 'c,d'])
```

Comment:
There is `\\` in test, but thats because you need to escape `\` with another `\`. When you write this in HTML attribute you can use just single `\` (tested).

I wanted to write a test where I would create an HTML element and read the value from there to demonstrate it, but to write `\` into the attribute I would need to write it as `\\` anyway :D

Reply:
yeah, def looks weird :d